ios/Podfile

```
@@ -0,0 +1,64 @@
+  post_install do |installer|
+    react_native_post_install(
+      installer,
+      config[:reactNativePath],
+      :mac_catalyst_enabled => false,
+      :ccache_enabled => podfile_properties['apple.ccacheEnabled'] == 'true',
+    )
+
+    # This is necessary for Xcode 14, because it signs resource bundles by default
+    # when building for devices.
+    installer.target_installation_results.pod_target_installation_results
+      .each do |pod_name, target_installation_result|
```

RuboCop (1.76.1)

[warning] Unused block argument - `pod_name`. If it's necessary, use `_` or `_pod_name` as an argument name to indicate that it won't be used.

(Lint/UnusedBlockArgument)